packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.controller.js

```diff
@@ -204,6 +204,15 @@ export default class {
     }
   }
 
+  trackKarapace(event) {
+    if (event.target === 'a') {
```

Comment:
event.target refers to html tag abject right?

Reply:
Yes. Corrected it.